sdk/options.go

```diff
@@ -61,12 +61,18 @@ func (o *Options) init() error {
 		o.ConsoleLogger = l
 	}
 
-	bs, err := ioutil.ReadAll(o.Config)
-	if err != nil {
-		return err
+	var bs = []byte("{}")
+
+	if o.Config == nil {
+		o.config = bs
```

Comment:
Also a nit: I think we could inline the value of `bs` here, as there is really no need for it in the outer scope
```suggestion
		o.config = []byte("{}")
```

Reply:
I actually like this one better. Making that adjustment now